themes/gauge_theme/layout.html

```diff
@@ -164,37 +146,63 @@
 {%- endblock %}
 
 {%- block footer %}
-  <footer>
-    <div class="powered-by">
-      <div class="container">
-        <a href="https://www.thoughtworks.com/products">
-          <img alt="Thoughtworks logo" src="{{ pathto('_static/img/thoughtworks-logo.svg', 1) }}" height="44">
-        </a>
-        <div><a href="browse.html">Table of Contents</a></div>
-        <div class="disclamer">
-          Gauge is an open source project, sponsored by ThoughtWorks Inc. under the GPL License, Version 3.0
+<footer>
+  <div class="footer container">
+    <div class="footer_items product">
+      <h5>Product</h5>
+      <nav class="footer_links">
+        <ul class="foolinks">
+          <li> <a href="get_started/">Get started</a> </li>
+          <li> <a href="/index.html#features">Features</a> </li>
+          <li> <a href="plugins/">Plugins</a> </li>
+          <li> <a href="//docs.gauge.org/latest/index.html" target="_blank">Documentation</a> </li>
+          <li> <a href="//blog.getgauge.io/"  target="_blank">Blog</a></li> 
+          <li> <a href="//brandfolder.com/twproducts"   target="_blank">Press Kit</a></li> 
+                      
+        </ul>
+      </nav>
+    </div>
+    <div class="footer_items explore">
+      <h5>Explore</h5>
+      <nav class="footer_links">
+        <ul class="foolinks">
+          <li> <a href="//waffle.io/getgauge/gauge"   target="_blank">Roadmap</a> </li>
+          <li> <a href="//manpage.gauge.org/"   target="_blank">Man page</a> </li>
+          <li> <a href="//insights.gauge.org/"   target="_blank">Insights</a> </li>
+        
+        </ul>
+      </nav>
+    </div>      
+    <div class="footer_items engage">
+      <h5>Engage</h5>
+      <nav class="footer_links">
+        <ul class="foolinks">
+          <li> <a href="//gitter.im/getgauge/chat" target="_blank"><img src="//preview.gauge.org/assets/images/footer-icons/gitter-icon.svg" class="icon-sm">Gitter</a> </li> 
+          <li> <a href="//groups.google.com/forum/#!forum/getgauge"  target="_blank"><img src="//preview.gauge.org/assets/images/footer-icons/google-group-icon.svg" class="icon-sm">Google Group</a> </li>
+          <li> <a href="//stackoverflow.com/users/login?ssrc=anon_ask&returnurl=https%3a%2f%2fstackoverflow.com%2fquestions%2fask%3ftags%3dgetgauge"  target="_blank"><img src="//preview.gauge.org/assets/images/footer-icons/stack_overflow-icon.svg" class="icon-sm">Stack Overflow</a> </li>
```

Comment:
the stackoverfflow link can he `https://stackoverflow.com/questions/tagged/getgauge`. Ref: https://github.com/getgauge/gauge.org/issues/11

Reply:
To keep it consistent to other link urls it is without https://.